cmd/simulator/simulate/loader/logtrigger.go

```diff
@@ -0,0 +1,66 @@
+package loader
+
+import (
+	"sync"
+
+	"github.com/smartcontractkit/ocr2keepers/cmd/simulator/config"
+	"github.com/smartcontractkit/ocr2keepers/cmd/simulator/simulate/chain"
+)
+
+// LogTriggerLoader ...
+type LogTriggerLoader struct {
+	// provided dependencies
+	progress ProgressTelemetry
+
+	// internal state values
+	mu       sync.RWMutex
+	triggers map[string][]interface{}
```

Comment:
Why interface rather than a concrete struct?

Reply:
Probably just a leftover holding place. I'll make it a concrete type.